<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncInsertDialog</class>
 <widget class="QDialog" name="IncInsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>承保公司编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>承保公司名称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_NumberEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Commit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>承保公司录入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>承保公司描述:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Reset">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>400</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Return">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>400</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_NameEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_DescriptionEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>371</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>801</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_11</zorder>
  <zorder>Insert_NumberEdit</zorder>
  <zorder>Insert_Commit</zorder>
  <zorder>label_10</zorder>
  <zorder>label_3</zorder>
  <zorder>Insert_Reset</zorder>
  <zorder>Insert_Return</zorder>
  <zorder>Insert_NameEdit</zorder>
  <zorder>Insert_DescriptionEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
